toolbox/build/venv-requirements.txt

```diff
@@ -1,5 +1,5 @@
 # runtime
-ansible>=4.5.0,<4.6
+ansible>=2.9.1,<2.10
```

Comment:
This should not be a revert, this should be a change in the original commit, is a revert once the code is merged, and this is not merged yet. Otherwise LGTM.

Reply:
We should have only 2 commits, not 3.

Reply:
Done and PR title changed too